fusor-ember-cli/app/mixins/validated-input-mixin.js

```diff
@@ -53,9 +53,11 @@ export default Ember.Mixin.create({
     // this action is triggered on key-up. however, for password fields only, we don't want
     // to show validation errors after every key stroke unless 8 characters have been entered
     showValidationErrorsKeyUp() {
-      if (!(this.get('isPassword') && this.get('value.length') < 8)) {
-        this.set("showValidationError", true);
-      }
+      let self = this;
+      // delay showing validation for 1 second
+      Ember.run.later((function() {
+        self.set("showValidationError", true);
+      }), 2000);
```

Comment:
@cfchase, that makes sense for key-down to cancel the timer and key-up to trigger it. It seems like a lot of triggers, but I think Ember is good at handling all this run loops.